app/views/install/compose.phtml

```diff
@@ -146,6 +146,7 @@ services:
       - _APP_STATSD_HOST
       - _APP_STATSD_PORT
```

Comment:
We need to add the `_APP_MAINTENANCE_RETENTION_CACHE` variable to all the containers that need it. 

It includes the `appwrite` and `appwrite-maintenance` containers